scripts/pc2sandbox_interactive.sh

```diff
@@ -38,7 +38,13 @@ FAIL_INTERACTIVE_ERROR=$((FAIL_RETCODE_BASE+55))
 MAXPROCS=32
 
 # taskset cpu mask for running submission on single processor
-CPUMASK=0x01
+cpunum=${USER/judge/}
+if [[ "$cpunum" =~ ^[1-5]$ ]]
```

Comment:
doesn't this restrict one use judge numbers 1 to 5 ?

Reply:
It does.  it should probably be much smarter and figure out how many CPUS's are available and only use the number that are.  This was more for testing on the same system though, to skew across the cpus.  In practice we use 'ubuntu' which pins to cpu 0 (default) and only one per system.